src/components/common/TmSessionSignUp.vue

```diff
@@ -107,6 +107,10 @@ export default {
       set(value) {
         this.$store.commit(`updateField`, { field: `signUpName`, value })
       }
+    },
+    isTestnet() {
+      // this.networks is undefined here O.o why??
+      return this.networks.filter(({ id }) => id === this.network)[0].testnet
```

Comment:
```suggestion
      return this.networks.find(({ id }) => id === this.network).testnet
```

Reply:
don't really know why this.networks is undefined tough

Reply:
yes, I know. But your solution works. I don't understand it either 🤷‍♀